presto-docs/src/main/sphinx/release/release-0.286.rst

```diff
@@ -0,0 +1,88 @@
+* Fix a potential bug in EXCEPT and INTERSECT queries. Do not prune unreferenced output in intersect and except nodes. :pr:`21343`
+* Fix bug of pull lambda out optimizers when expressions pulled out are of type JoniRegexType or LikePatternType.
+* Fix lambda expression pull out optimizer so that it will be able to extract independent expressions from conditional expression. :pr:`21344`
+* Fix serialized name for retriable QueryError property.
+* Fix the compilation error of merge function for KHyperLogLog state.
+* Fixed a bug in cte materialization which was causing incorrect plan generation in some cases.
+* Fixed redis-provider-plugin docs which previously contained an incorrect file name for property loading, and included the HBO configurations for the coordinator.
+* Fixed writtenIntermediateBytes metric by ensuring its correct population through temporary table writers.
+* Improve efficiency of partial aggregation step. This can be enabled by setting the ``adaptive_partial_aggregation`` session property to true.
+* Improve predicate pushdown through Joins.
+* Improve the readability of query plan by making the number printed in query plan to be comma separated.
+* Add REGR_AVGX, REGR_AVGY, REGR_COUNT, REGR_R2, REGR_SXX, REGR_SXY, and REGR_SYY.
+* Add UPDATE sql support in Presto Engine.
+* Add a session property to skip hash precomputation for join when the input is table scan, and the hash is on a single big int and is not reused later. It's controlled by session property `skip_hash_generation_for_join_with_table_scan_input` and default to not enabled.
+* Add an feature config property `khyperloglog-agg-group-limit` to set the maximum number of groups `khyperloglog_agg` can have. It will fail the query when the limit is exceeded. The default is 0 which means no limit.
+* Add an feature config property `limit-khyperloglog-agg-group-number-enabled` to control whether to limit the number of groups for aggregation functions using KHyperLogLog state.
+* Add an optimization to optimize queries which has equivalence check filter or constant assignments. Controlled by session property `rewrite_expression_with_constant_expression` and default to enabled.
+* Add an optimization to rewrite left join with array contains in join condition to equi join, the optimization is controlled by session property `rewrite_left_join_array_contains_to_equi_join `.
+* Add client option to disable redirects.
+* Add simple configurations to prestodb/presto docker image and make the image easier to use, including ``config.properties`` and ``jvm.config``.
+* Add support for running on Linux and M1/M2 macOS aarch64 (ARM64).
+* Add validation to check that next URI host and port does not change during query execution.
+* Added 1 folder(ecosystem) and 1 file in sphinx.
+* Added 1 line in sphinx/index.rst.
+* Added CTE hash partition count session property.
```

Comment:
* Add cte_hash_partition_count session property to specify the number of buckets or writers to be used when using CTE Materialization :pr:`21625`